src/screens/authentication/css/authentication.css

```diff
@@ -21,175 +20,326 @@
+  font-size: 50px;
+  font-weight: 800;
+}
+.splash-text-heading > span {
+  color: #6f3fe3;
+}
+.splash-subtitle-text {
+  color: #e4edf6;
+  font-family: Poppins-regular;
+  font-size: 18px;
+  font-weight: 500;
+}
+
+/* login page */
+.light-signup-container 
+.light-login-text {
+  color: #0272bc;
+  font-family: Poppins-bold;
+  font-size: 26px;
+  font-weight: 700;
+}
+.dark-login-text {
+  color: #048ad7;
+  font-family: Poppins-bold;
+  font-size: 26px;
+  font-weight: 700;
+}
+.light-login-label-texts {
+  color: #585858;
+  font-family: Poppins-regular;
+  font-size: 16px;
+  font-weight: 700;
+}
+.dark-login-label-text {
+  color: #fffcfc;
+  font-family: Poppins-regular;
+  font-size: 16px;
+  font-weight: 700;
+}
+.light-login-subtitle-text {
+  color: #585858;
+  font-family: Poppins-medium;
+  font-size: 14px;
+  font-weight: 400;
+}
+.dark-login-subtitle-text {
+  color: #fff;
+  font-family: Poppins-medium;
+  font-size: 14px;
+  font-weight: 400;
+}
+.light-login-subtitle-text > a {
+  color: #150aa1;
+  font-weight: 600;
+}
+.dark-login-subtitle-text > a {
+  color: #048ad7;
+  font-weight: 600;
+}
+.login-option {
+  color: #7b7b7b;
+  font-family: Poppins-regular;
+  font-size: 14px;
+  font-weight: 400;
+}
+.login-option > span {
+  color: #595959;
+  font-size: 16px;
+  font-weight: 500;
+}
+.logUp-btn-texts {
+  color: #7b7b7b;
+  font-family: Poppins-medium;
+  font-size: 14px;
+  font-weight: 400;
+}
+.light-reset-password-subtitle-text-mode {
+  color: #000;
+  font-family: Poppins;
+  font-size: 14px;
+  font-weight: 400;
+  text-align: center;
+}
+.dark-reset-password-subtitle-text-mode {
+  color: rgba(255, 252, 252, 0.9);
+  font-family: Poppins;
+  font-size: 14px;
+  font-weight: 400;
+  text-align: center;
+}
+.password-input-text {
+  color: #a9a9a9;
+  font-family: Poppins-medium;
+  font-size: 14px;
+  font-weight: 400;
+  /* padding-left: 0.5rem; */
+}
+
+
+/* ----------------------------------optimazing login page styles //Osita ------------------------- */
+
+
+
+/*  ----------login password page--------------------  */
+.light-signup-container >
+.light-signup-form-wrapper >
+.login-form
+{
+  gap: 0.5rem;
+  position: relative;
+}
+.password {
+  position: relative;
+
+}
+
+.light-signup-form-wrapper
+.login-input {
+  width: 388px;
+  height: 51px;
+  border-radius: 10px;
+  border: 1px solid #E4E4E4;
+  background: #F1F1F1;
+  padding-left: 0.5rem;
+}
+.light-signup-form-wrapper
+.password
+.open-eye-icon {
+  width: 19px;
+  height: 13px;
+  position: absolute;
+  right: 11.55px;
+  bottom: 19.5px;
+}
+.login-password-subtitle {
+  /* margin-left: 15%;
+  margin-right: 25%; */
+  width: 300px;
+  box-sizing: border-box;
+  height: auto;
+  /* background: #000; */
+  /* height: 21px; */
+  display: flex;
+  /* justify-content: space-around; */
+  /* margin: auto;
+  display: flex;
+  margin-top: 1rem; */
+  /* display: flex; 
+  align-items: flex-start; */
+  /* gap: 100px; */
+
```

Comment:
there was part of the code i told him to commit because he was not getting it and i asked him to comment it so i can check and work on them. i believe those are the part commented

Reply:
Oh okay. I guess it’s settled then.